docs/administrator_guide/enrichment/dynamic_searching.md

```diff
@@ -0,0 +1,76 @@
+
+  ![](media/dynamic_searching/DynamicSearchingConfigurationSetup.PNG)
+
+### Dynamic Search Configuration Parameters
+
+`Dynamic Searching Object Types Json` field is inputted as JSON with each `{}` representing a single object type and field in that object type.
+
+**Parameters of `Dynamic Searching Object Types Json`:**
+- `ObjectTypeName` - [Required] an object type for which search will be performed
+- `FieldName` - [NotRequired] a field which value will be used to perform search (by default it is set to `Identifier`)
+
+
+**Example `Dynamic Searching Object Types Json` configuration:**
+```json
+[
+   {
+      "ObjectTypeName":"Financial Products",
+      "FieldName":"Identifier"
+   },
+   {
+      "ObjectTypeName":"Pharmaceutical Product"
+   }
+]
+```
+
+> **`Dynamic Searching Object Types Json` Validation Rules:**
+>
+> - `ObjectTypeName` must be a name of object type which exists in the workspace
+> - One object type can occur only once in `Dynamic Searching Object Types Json` configuration
+> - `FieldName` must be a field which exists for gived object type, also it must be a fixed-length field which is an object type's identifer
+> - Object type can't be saved in configuration if there are no RDOs of gived object type
+> - Object type can't be saved in configuration if there are RDOs of this object type with duplicated values in identifer field
+
+## Dynamic Searching Execution
+
+After saving `Dynamic Searching Object Types Json` a new configuration is created automatically for every object type + field pair added to json.
+1) New multiple object field called `ObjectTypeName` + `FieldName` is created on `Document` object type. This field joins `Document` and particular `Object Type`.
+
+    ![](media/dynamic_searching/DynamicSearching_MultipleObjectDocumentField.PNG)
+
+2) New `Persistent Highlight Set` is created and it is set to highlight values from multiple object document field created in previous step.
+
+    ![](media/dynamic_searching/DynamicSearching_PersistentHighlightSet.PNG)
+
+3) New `Data Mapping` is created for multiple object document field created in first step. This data mapping is also added to each `Ingestion Profile` set up for `Document` object type.
+
+    ![](media/dynamic_searching/DynamicSearching_DataMapping.PNG)
+
+When new documents are ingested into the system and data transformations are executed on documents' data batch, documents are tagged with objects set up in `Dynamic Searching Object Types Json` setting.
+
+![](media/dynamic_searching/DynamicSearching_DocumentView.PNG)
+![](media/dynamic_searching/DynamicSearching_HighlightedDocument.PNG)
+
+## Dynamic Searching Limitations
+1) Objects of object type set up in `Dynamic Searching Object Types Json` cannot contain semicolon (`;`) in identifier field.
+2) `Insight.Shared.Toggles.RunTraceDynamicSearchTransformation` toggle must be set to true in order for Dynamic Search to be performed.
+
+
```

Comment:
Could we add in a picture of how the objects are stored in the workspace? 

![image](https://user-images.githubusercontent.com/91223946/179097006-9647990a-c2c2-4559-a123-90e9370ff7be.png)

Reply:
Hi @SoniaChy ! I've changed the documentation (added Financial Identifiers view and removed pard about FieldName being configurable)